### DISCUSSION TITLE
Allow a setting to allow live and pending entries to be shown on form submissions when element connections are used

### DISCUSSION THREAD

**Discussion by terryupton** [solspace/craft-freeform#664] [Feature Requests]
It would be good to have a setting (or atomically) to allow pending entries and potentially disabled entries to show on form submissions when element connections are used in the form. Currently, this only shows live entries and has cased us some issues in some situations.

**Comment by kjmartens** [comment]
This has been noted. Thanks for your feedback @terryupton 🙂

**Comment by kjmartens** [comment]
We've now added a "Status" filter in Freeform 5.12+ when populating form fields with Element data. You can choose to include 1 or more, or all statuses in the results. 🙂 

https://docs.solspace.com/craft/freeform/v5/forms/fields/#populating-field-options
